modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/GoClientCodegen.java

```diff
@@ -97,7 +97,7 @@ public GoClientCodegen() {
         typeMapping.put("boolean", "bool");
         typeMapping.put("string", "string");
         typeMapping.put("UUID", "string");
-        typeMapping.put("date", "time.Time");
+        typeMapping.put("date", "SwaggerDateType");
```

Comment:
@antihax Shall we keep `time.time` so that the auto-generated code at least compile?